GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.cs

```diff
@@ -14,10 +14,16 @@ namespace GitUI.CommandsDialogs.SettingsDialog.Pages
 {
     public partial class AppearanceSettingsPage : SettingsPageWithHeader
     {
+        private const string customAvatarTemplateURL = "https://git-extensions-documentation.readthedocs.io/en/latest/settings.html#git-extensions-appearance-author-images";
+        private const string gravatarDefaultImageURL = "http://en.gravatar.com/site/implement/images#default-image";
```

Comment:
This link should point to the GE documentation too, it does not cover Initials
It should also describe that Initials is local generated

Reply:
✔ done